src/web/js/trove/chart-lib.js

```diff
@@ -1253,10 +1254,16 @@
             } else {
               labelRow = '';
             }
+            let labelRowY = null;
```

Comment:
I wouldn't overload this condition right now -- I would much rather you create an entirely separate condition for dot plots!

For now there will be a lot of duplicate logic (the first 5-10 LOC), but this can all be abstracted out once `ps` and `tab` are unified.